src/main/resources/db/migration/V1_94_1__referral_details_table.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE referral_details(
+    id uuid not null,
+    superseded_by_id uuid unique,
+    created_at timestamp with time zone not null,
+    created_by text not null,
+    reason_for_change text,
+    referral_id uuid not null,
+    completion_deadline date,
+    further_information text,
+    maximum_enforceable_days int,
+
+    primary key(id),
+    foreign key (superseded_by_id) references referral_details(id)
+);
+
```

Comment:
Are these metadata values correct. Further information was classes as sensitive on the referral table, so that was copied across. All others I set to false.